README.md

```diff
@@ -124,6 +124,15 @@ docker run -it buoyantio/slow_cooker -qps 100 -concurrency 10 http://$(docker-ma
 docker build -t buoyantio/slow_cooker -f Dockerfile .
 ```
 
+### Build Multi-Arch Images with Buildx
```

Comment:
Can you add a note and commands here to create the multiarch builder image? Something like what we have here: https://github.com/linkerd/linkerd2-proxy-init/blob/master/Makefile#L73